Cargo.toml

```diff
@@ -45,7 +45,8 @@ mesalink = ["curl-sys/mesalink"] # MesaLink TLS backend
 http2 = ["curl-sys/http2"]
 spnego = ["curl-sys/spnego"]
 rustls = ["curl-sys/rustls"]
-static-curl = ["curl-sys/static-curl"]
+static-curl = ["curl-sys/static-curl", "curl-sys/zlib"]
+static-curl-no-zlib = ["curl-sys/static-curl"]
```

Comment:
I don't think that a new `static-curl-no-zlib` feature is necessary, could a new `zlib` feature be added which is added to `default`?

Reply:
Do you mean there should be a `zlib` feature here, which is included in the `default` set, but only takes effect when `static-curl` is turned on.
If we want to disable `zlib`, we need to set `default-features` to `false` like this:

```toml
[dependencies]
curl = { version = "*", default-features = false, features = ["static-curl"] }
```

Reply:
Technically moving a feature that used to always be available behind a feature under `default` is a breaking change. Any crate which was already setting `default-features = false` and expecting zlib to be included when doing will suddenly _not_ have zlib support.